tests/test_plugin.py

```diff
@@ -73,27 +75,27 @@ class TestSplitToSuites:
                 1,
                 2,
                 [
-                    ["test_1", "test_2", "test_3", "test_4", "test_5", "test_6", "test_7"],
-                    ["test_8", "test_9", "test_10"],
+                    ["test_1", "test_3", "test_5", "test_7", "test_9"],
```

Comment:
One of the key features of `pytest-split` is that it maintains the order of the tests. It's one of the hard requirements at least in one of my own use cases.

Thus, we should either
* provide a `--maintain-order` flag or similar
* provide a `--split-algorithm` argument which we can use for different algorithms. We could use the current one (which maintains the order) as a default and come up with some fancy name for this new one.

Reply:
I knew for you order was important, that's why I made sure the _relative_ order of tests is not changed.
But if I understand you correctly, you what the order of test of group 1/3, then group 2/3 and afterwards group 3/3 to be the same as without invoking this plugin.

I think adding `--split-algorithm` makes the most sense. I'll add some documentation about the properties of the various algorithms.

Reply:
yeah, if we consider this test case, I basically need to have order `test_1`, `test_2`, `test_3` .... and the split can happen at any point but e.g. it's not ok that we have `test_1` and `test_3` in group1 while `test_2` in group2

Reply:
@jerry-git this should be ready for a second review.